### DISCUSSION TITLE
New link from NE

### DISCUSSION THREAD

**Discussion by Swati-Dash** [digital-land/data-standards-backlog#95] [General]
site-of-special-scientific-interest dataset endpoint from the Natural England ArcGIS plugin has a couple of warnings.

It appears that during the stage where ArcGIS plugins are processed, a read timeout exception occurs while executing the for loop in the[ arcgis.py file](https://github.com/digital-land/digital-land-python/blob/main/digital_land/plugins/arcgis.py). 
Hence the encoding part is not executed, which leads to a second warning.

Can you pls investigate this issue and identify if its a server side issue so that we can add this to the platform

Endpoint URL: [Layer: Sites of Special Scientific Interest (England) © Natural England (ID:0)](https://services.arcgis.com/JJzESW51TqeY9uat/arcgis/rest/services/SSSI_England/FeatureServer/0) 
Dataset: site-of-special-scientific-interest

**Comment by colmjude** [comment]
Hi, @Swati-Dash; we were able to download and process this file (see [analysis notes](https://github.com/digital-land/data-standards-backlog/discussions/96#discussioncomment-11210628)).
@eveleighoj could there be an issue in the plugin code?
